### PR TITLE
Fix Typo - "TokenAccessReviewStatus" -> "status"

### PR DESCRIPTION
**Pull request by dims**
"TokenAccessReview" is the value of the "kind" field of the request and response. The webhook has to fill in the "status" field to indicate authenticate pass/fail status.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4475)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ The request body will be of the following format:
 }
 ```
 
-The remote service is expected to fill the `TokenAccessReviewStatus` field of
+The remote service is expected to fill the `status` field of
 the request to indicate the success of the login. The response body's "spec"
```
Comment (ericchiang) [inline docs/admin/authentication.md]: For consistency, can you update
```
"spec" -> `spec`
```

Comment (dims) [inline docs/admin/authentication.md]: Done

**Comment by ericchiang** [comment]
Thanks!

**Pull request merged**